tests/testthat/testNonASCII/R/a.r

```diff
@@ -0,0 +1,9 @@
+# This script is intended to be saved in GB2312 to test if non UTF-8 encoding is
+# supported.
+
+#' ����ע��
```

Comment:
The fact that this previews correctly makes me worried that it isn't actually saved as GB2312.